sdk/core/AzureCore/Source/Pipeline/Policies/AuthenticationPolicy.swift

```diff
@@ -24,16 +24,21 @@
 //
 // --------------------------------------------------------------------------
 import Foundation
-import MSAL
 
 public typealias TokenCompletionHandler = (AccessToken?, Error?) -> Void
-public typealias MSALResultCompletionHandler = (MSALResult?, Error?) -> Void
 
 public struct AccessToken {
     // MARK: Properties
 
     public let token: String
     public let expiresOn: Int
```

Comment:
Per guidelines, should this be a Date that is internally converted to Int?

Reply:
Yep, good call :)